hazelcast/src/main/java/com/hazelcast/sql/impl/extract/QueryPath.java

```diff
@@ -23,23 +23,24 @@
 import com.hazelcast.sql.impl.SqlDataSerializerHook;
 
 import java.io.IOException;
+import java.io.Serializable;
 
 import static com.hazelcast.query.QueryConstants.KEY_ATTRIBUTE_NAME;
 import static com.hazelcast.query.QueryConstants.THIS_ATTRIBUTE_NAME;
 
 /**
  * Represent a path to the attribute within a key-value pair.
  */
-public final class QueryPath implements IdentifiedDataSerializable {
+public final class QueryPath implements IdentifiedDataSerializable, Serializable {
```

Comment:
The same question as in the `Expression` class - do we need this `Serializable` marker still?